grafana-plugin/src/models/cloud/cloud.ts

```diff
@@ -49,12 +49,12 @@ export class CloudStore extends BaseStore {
     });
   }
 
-  getSearchResult() {
+  getSearchResult = () => {
```

Comment:
AFAIR you were going to move this outside of mobx store, or it will be done in the next PRs?

Reply:
In this PR I've done it for alert_receive_channel only, otherwise there will even more changes in PR which I wanted to avoid. In subsequent PR I'll use autogenerate types for alertgroups and user stores and I'll do the same for them. The rest can be covered by https://github.com/grafana/oncall/issues/3734